README.adoc

```diff
@@ -8,42 +8,430 @@ The time for the full course is around three to four days.
 
 The material is created with people with zero Rust experience but with a programming background in mind.
 
-## Overview
+Ferrous Systems offers a large Rust curriculum for both beginner and
+advanced Rust developer at all levels. We are the oldest and most often
+chosen Rust training provider.
+
+Ferrous Systems specialises in custom, topic focused workshops for
+enterprises starting moving into Rust. Supplemental to our courses, we
+offer ongoing help and feedback.
```

Comment:
Link to Rust-Experts?

Reply:
If it's folded into a course offering, I would generally prefer to not mingle brands.